openedx/core/djangoapps/heartbeat/urls.py

```diff
@@ -1,10 +1,8 @@
 """
 Urls for verifying health (heartbeat) of the app.
 """
-from django.conf.urls import patterns, url
-
-urlpatterns = patterns(
-    '',
+from django.conf.urls import url
 
+urlpatterns = [
     url(r'^$', 'openedx.core.djangoapps.heartbeat.views.heartbeat', name='heartbeat'),
```

Comment:
Shouldn't this view name be converted to a callable?

Reply:
Yes indeed.